src/components/PaginationV2/PaginationV2.js

```diff
@@ -186,7 +185,7 @@ export default class PaginationV2 extends Component {
                 )}
           </span>
         </div>
-        <div className="bx--pagination__right bx--pagination--inline">
+        <div className="bx--pagination__right">
```

Comment:
Is the `bx--pagination--inline` class not needed anymore in this component?

Reply:
it's just that there is a slight shift of the number in using it, it's not centred, but it can be used

Reply:
yes, I can change this